src/workerd/api/queue.h

```diff
@@ -32,30 +32,48 @@ class WorkerQueue: public jsg::Object {
     // contentType determines the serialization format of the message.
     jsg::Optional<kj::String> contentType;
 
-    JSG_STRUCT(contentType);
+    // The number of seconds to delay the delivery of the message being sent.
+    jsg::Optional<uint> delaySecs;
+
+    JSG_STRUCT(contentType, delaySecs);
     JSG_STRUCT_TS_OVERRIDE(QueueSendOptions {
       contentType?: QueueContentType;
+      delaySecs ?: number;
```

Comment:
I don't imagine the override is needed here given that we aren't change the type at all. `contentType` only needed an override due to wanting to advertise that it only accepts the `QueueContentType` type

But you should be able to confirm by building the types locally using the instructions in https://github.com/cloudflare/workerd/tree/main/types

Reply:
Did you manage to get this building to see how the resulting types look? My machine unfortunately complains about a dependency issue when I try, otherwise I'd do it for you.

I'm fine skipping it for now and just validating them after merge if needed, so long as you do come back and make sure the published types look ok afterwards.